README.md

```diff
@@ -17,3 +17,35 @@ a few minutes and will help us better support the Blockly community.
 Cross-browser Testing Platform and Open Source <3 Provided by [Sauce Labs](https://saucelabs.com)
 
 Want to contribute? Great! First, read [our guidelines for contributors](https://developers.google.com/blockly/guides/modify/contributing).
+
+## Releases
+
+We release by pushing the latest code to the master branch, followed by updating our [docs](developers.google.com/blockly) and [demo pages](https://blockly-demo.appspot.com). We typically release a new version of Blockly once a quarter (every 3 months). If there are breaking bugs, such as a crash when performing a standard action or a rendering issue that makes Blockly unusable, we will cherry-pick fixes to master between releases to fix them.
+
+Releases are tagged by the release date (YYYYMMDD) with a leading '1.' and a trailing '.0' in case we ever need a major or minor version. For example, [1.20190419.0](https://github.com/google/blockly/tree/1.20190419.0)
```

Comment:
Add a link to the releases page: https://github.com/google/blockly/releases

Reply:
Also in this section, maybe add that you can use specific releases directly from github with npm: `npm install git://github.com/google/blockly.git#1.20181219.0`